sunpy/util/exceptions.py

```diff
@@ -12,6 +12,7 @@ class SunpyWarning(AstropyWarning):
     The base warning class from which all Sunpy warnings should inherit.
 
     Any warning inheriting from this class is handled by the Sunpy logger.
+    Should likely not be used.
```

Comment:
eh?

Reply:
This is the base class. People should be using `SunpyUserWarning` which is also true for normal Python. The default is `UserWarning` not `Warning`.

Reply:
Can you add that to the docstring? If I didn't get it others probably wont either.

Reply:
Added better language.